src/kernel/arch/x86/paging.zig

```diff
@@ -99,10 +99,10 @@ const TENTRY_AVAILABLE: u32 = 0xE00;
 const TENTRY_PAGE_ADDR: u32 = 0xFFFFF000;
 
 /// The number of bytes in 4MB
-pub const PAGE_SIZE_4MB: u32 = 0x400000;
+pub const PAGE_SIZE_4MB: usize = 0x400000;
 
 /// The number of bytes in 4KB
-pub const PAGE_SIZE_4KB: u32 = PAGE_SIZE_4MB / 1024;
+pub const PAGE_SIZE_4KB: usize = PAGE_SIZE_4MB / 1024;
```

Comment:
Do these need to be `usize` as this in the x86

Reply:
I guess it doesn't need to be for the reasons I listed in the issue, but I think that something describing a size should be `usize` anyway. It also means casting is unnecessary when using it as the block size for the vmm and pmm.